.ci_support/osx_64_python3.8.____cpython.yaml

```diff
@@ -3,13 +3,13 @@ MACOSX_DEPLOYMENT_TARGET:
 c_compiler:
 - clang
 c_compiler_version:
-- '12'
+- '13'
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 llvm_openmp:
-- '12'
+- '13'
```

Comment:
I've found OpenMP 12+ to be buggy on MacOS, see this commit https://github.com/pace-neutrons/Euphonic/tree/e7f966e1876317e221bd90acace99d612ccaf297. But I don't think this was an issue for us in the last release where it was version 12 so it should be fine? Just noting it here

Reply:
Shouldn't hurt to bump it like this